README.md

```
@@ -31,12 +31,24 @@
-> Open-source AI Agent that empowers data-driven teams to chat with their data to generate Text-to-SQL, charts, spreadsheets, reports, and BI. 
+> Open-source GenBI AI Agent that empowers data-driven teams to chat with their data to generate Text-to-SQL, charts, spreadsheets, reports, and BI. 
 
 <p align="center">
   <img src="./misc/wren_workflow.png">
 </p>
 
+## 🕶 Try it yourself!
+
+### GenBI (Generative Business Intelligence)
+
+https://github.com/user-attachments/assets/90ad1d35-bb1e-490b-9676-b29863ff090b
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)